python/hail/docs/scans.rst

```diff
@@ -5,13 +5,21 @@ Scans
 
 The ``scan`` module is exposed as ``hl.scan``, e.g. ``hl.scan.sum``.
 
-The functions in this module perform rolling aggregations along the desired axis;
+The functions in this module perform rolling aggregations along the rows of a
+table, or along the rows or columns of a matrix table. The value of the scan at
+a given row (or column) is the result of applying the corresponding aggregator
+to all previous rows . Scans directly over entries are not currently supported.
+
 For example, the ``count`` aggregator can be used as ``hl.scan.count`` to add an
-index along the rows of a `Table` or the rows or columns of a `MatrixTable`.
+index along the rows of a `Table` or the rows or columns of a `MatrixTable`; the
```

Comment:
I prefer the convention of "table" and "matrix table" in doc notes as referring to our core relational notions, unless specifically trying to emphasize the class.